pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1182,6 +1212,11 @@ func (b *MasqueradeBindMechanism) createNatRulesUsingNftables(proto iptables.Pro
 		return err
 	}
 
+	err = b.skipForwardingForReservedPorts(proto)
+	if err != nil {
```

Comment:
You can inline this.

Reply:
I prefer to keep the style uniform (eg. if you see the preceding lines)